src/charm.py

```diff
@@ -141,6 +149,26 @@ def __init__(self, *args):
 
         self.log_forwarder = LogForwarder(self)
 
+        self.loki_device_alert_rules_path = "./loki_alert_rules"
+        self.loki_device_alerts_push_api_consumer = LokiPushApiConsumer(
```

Comment:
Please rename this `loki_push_api_consumer_devices` to follow the pattern somewhat established

Reply:
Is this object also automatically pushing logs? If so, which logs ? The charm's log ? The payload's logs ? I'm trying to understand if that's a duplicate to the `log_forwarder` above.

Reply:
It's not pushing logs by itself, it's instead exposing the loki_endpoints (i.e: http://loki:3100/loki/api/v1/push).
This is meant for you to push things there. You could push your own logs there but this would require extra work to match the Loki API.